2.5/README.md

```diff
@@ -0,0 +1,141 @@
+Ruby 2.5 Docker image
+=================
+
+This container image includes Ruby 2.5 as a [S2I](https://github.com/openshift/source-to-image) base image for your Ruby 2.5 applications.
+Users can choose between RHEL and CentOS based builder images.
+The RHEL image is available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/ruby-25-rhel7)
+as registry.access.redhat.com/rhscl/ruby-25-rhel7.
+The CentOS image is then available on [Docker Hub](https://hub.docker.com/r/centos/ruby-25-centos7/)
+as centos/ruby-25-centos7.
+The resulting image can be run using [Docker](http://docker.io).
+
+Description
+-----------
+
+Ruby 2.5 available as docker container is a base platform for
+building and running various Ruby 2.5 applications and frameworks.
+Ruby is the interpreted scripting language for quick and easy object-oriented programming.
+It has many features to process text files and to do system management tasks (as in Perl).
+It is simple, straight-forward, and extensible.
+
+Usage
+---------------------
+To build a simple [ruby-sample-app](https://github.com/sclorg/s2i-ruby-container/tree/master/2.5/test/puma-test-app) application
```

Comment:
The linked URI does not exist yet. Would you mind creating PR for it?

Reply:
What do you mean?
We should test below one before this PR will be merged?

```
$ s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/ rhscl/ruby-25-rhel7 ruby-sample-ap
```

Reply:
See the EDIT, please.